test/functionalTest/cases/1367_time_measures/time_measure_stats.test

```diff
@@ -217,7 +217,7 @@ echo
 01. Second call to GET /statistics (gives time stat of FIRST request for /statistics)
 =====================================================================================
 HTTP/1.1 200 OK
-Content-Length: 127
```

Comment:
I was having problems with Content-Length of statistics and metrics text. For instance, I see that 0.71234 changed to 0.6223 which stills matching the REGEX() for that value, but reduce the Content-Length in 1.

In general, Content-Length in statistics/metrics test is not very important. In fact, I have already seen many `Content-Length: REGEX(.*)` and `Content-Length: REGEX(\d+)`. Thus, I understand the similar changes in this PR are ok.

Reply:
yeah, no reason to include the exact Content-Length. the payload is there to be compared. more than enough.

NTC